app/helpers/days_helper.rb

```diff
@@ -19,4 +19,10 @@ def day_status_css(d)
   def feedback_submitted?(day)
     student? && current_user.day_feedbacks.find_by(day: day.to_s)
   end
+
+  def completed_students(activity)
+    completed = cohort.students.select { |student| student.completed_activity?(activity) }.count
```

Comment:
Can you think of a way to do this in a single database query instead of a separate query for each student for each assignment? Would it make sense for that query to be a scope in Student or Activity? E.g. `cohort.students.completed_activity(activity).count`.